pylutron_caseta/smartbridge.py

```diff
@@ -565,6 +566,37 @@ def _handle_occupancy_group_status(self, response: Response):
             if occgroup_id in self._occupancy_subscribers:
                 self._occupancy_subscribers[occgroup_id]()
 
+    def _handle_ra3_occupancy_group_status(self, response: Response):
+        _LOG.debug("Handling ra3 occupancy status: %s", response)
+
+        if response.Body is None:
+            return
+
+        statuses = response.Body.get("AreaStatuses", {})
```

Comment:
```suggestion
        statuses = response.Body.get("AreaStatuses", [])
```
This looks like it's supposed to be a list, not a dictionary.